### PR TITLE
[C-2961] Fix LeftNav SEO

### PR DESCRIPTION
**Pull request by dylanjeffers**
### Description

Major improvements to left nav seo.
1. Adds link for home-page
2. Adds link

### Dragons

*Is there anything the reviewer should be on the lookout for? Are there any dangerous changes?*

### How Has This Been Tested?

*Please describe the tests that you ran to verify your changes. Provide repro instructions & any configuration.*

### How will this change be monitored?

*For features that are critical or could fail silently please describe the monitoring/alerting being added.*

### Feature Flags ###

*Are all new features properly feature flagged? Describe added feature flags.*

### CODE REVIEW

File: packages/stems/src/components/Button/Button.module.css [hunk 1]
```diff
@@ -137,17 +136,17 @@
   fill: var(--primary-secondary-text);
 }
 
-.primary.includeHoverAnimations:hover:enabled {
```
Comment (dylanjeffers) [inline packages/stems/src/components/Button/Button.module.css:140]: headsup this is needed cause anchor tags dont have this pseudoelement, but can be generalized with attaching an "enabled" classname

File: packages/stems/src/types.ts [hunk 2]
```diff
@@ -0,0 +1,7 @@
+declare module 'react' {
```
Comment (dylanjeffers) [inline packages/stems/src/types.ts:1]: needed to allow generic forwardRefs

**Review by raymondjacobson — APPROVED** [review]
so thorough. thanks for the helper comments :)

File: packages/web/src/components/avatar/Avatar.tsx [hunk 3]
```diff
@@ -35,6 +36,8 @@ export const Avatar = (props: AvatarProps) => {
     SquareSizes.SIZE_150_BY_150
   )
 
+  const image = userId ? profileImage : imageProfilePicEmpty
```
[the remark this answers is not in the document]
Comment (dylanjeffers) [reply packages/web/src/components/avatar/Avatar.tsx:39]: yeah definitely possible in the future, but this was my quick and dirty way to get one avatar working. can update if there are cases of this not working though. my hope is the id should at least be there no matter what

File: packages/web/src/components/typography/typography.module.css [hunk 4]
```diff
@@ -207,6 +207,10 @@
   font-weight: var(--font-demi-bold);
 }
 
+.bodyWeak {
```
[the remark this answers is not in the document]
Comment (dylanjeffers) [reply packages/web/src/components/typography/typography.module.css:210]: LOL

**Pull request merged**